src/i18n/index.js

```diff
@@ -1,6 +1,6 @@
 import { messages as headerMessages } from '@edx/frontend-component-header';
 import { messages as footerMessages } from '@edx/frontend-component-footer';
```

Comment:
```suggestion
import { messages as headerMessages } from '@openedx/frontend-component-header';
import { messages as footerMessages } from '@openedx/frontend-component-footer';
```
I imagine these will also need to be updated, right?

Reply:
Ah, I misread the `package.json` changes, which was that only Paragon and Frontend Build are being switched to `@openedx` for this PR. Please disregard 😅